src/components/Basic/Modal/Modal.module.scss

```diff
@@ -0,0 +1,60 @@
+.modal {
+    position: absolute;
+    width: 100vw;
+    height: 100vh;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    pointer-events: none;
+
+    &-open {
+        pointer-events: auto;
+    }
+
+    &-overlay {
+        position: absolute;
+        width: 100%;
+        height: 100%;
+        background-color: rgba(0, 0, 0, 0.3);
+        animation: fadeIn 0.3s forwards;
+    }
+
+    &-window {
+        display: flex;
+        padding: 1vmax;
+        flex-direction: column;
+        position: absolute;
+        width: 40vmax;
+        height: 20vmax;

```

Comment:
```suggestion
        min-height: 20vmax;

```

It will fix this overflow: 
<img width="584" height="332" alt="image" src="https://github.com/user-attachments/assets/6809d243-d6f1-43e5-ad7c-f936a739dd61" />